From 722d4fc65987acdae57760c7d4bd82f4380e09c5 Mon Sep 17 00:00:00 2001
From: Ronny Lorenz <ronny@tbi.univie.ac.at>
Date: Tue, 8 Dec 2015 14:56:16 +0100
Subject: [PATCH 2/8] use chromiumos mwifiex drivers

Ported from changes by Kevin Mihelich applied to the ChromeOS 3.8 branch
---
 drivers/net/wireless-3.8/mwifiex/sdio.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/wireless-3.8/mwifiex/sdio.h b/drivers/net/wireless-3.8/mwifiex/sdio.h
index b4877f4..d366036 100644
--- a/drivers/net/wireless-3.8/mwifiex/sdio.h
+++ b/drivers/net/wireless-3.8/mwifiex/sdio.h
@@ -30,9 +30,9 @@
 #include "main.h"
 
 #define SD8786_DEFAULT_FW_NAME "mrvl/sd8786_uapsta.bin"
-#define SD8787_DEFAULT_FW_NAME "mrvl/sd8787_uapsta.bin"
-#define SD8797_DEFAULT_FW_NAME "mrvl/sd8797_uapsta.bin"
-#define SD8897_DEFAULT_FW_NAME "mrvl/sd8897_uapsta.bin"
+#define SD8787_DEFAULT_FW_NAME "mrvl/sd8787_uapsta_cros.bin"
+#define SD8797_DEFAULT_FW_NAME "mrvl/sd8797_uapsta_cros.bin"
+#define SD8897_DEFAULT_FW_NAME "mrvl/sd8897_uapsta_cros.bin"
 
 #define BLOCK_MODE	1
 #define BYTE_MODE	0
-- 
2.6.3

